mix.exs

```diff
@@ -36,10 +36,10 @@ defmodule Numbers.Mixfile do
     [
       {:coerce, "~> 1.0"},
 
-      {:decimal, "~> 1.9", optional: true},
+      {:decimal, "~> 2.0", optional: true},
 
       {:ex_doc, "~> 0.19", only: [:docs], runtime: false},
-      {:inch_ex, only: :docs}                  # Inch CI documentation quality test.
+      {:inch_ex, "~> 2.0.0", only: :docs}                  # Inch CI documentation quality test.
```

Comment:
Is there a reason to restrict the version here? It probably should not be part of this PR.